util/file.py

```diff
@@ -94,6 +94,27 @@ def mkstempfname(suffix='', prefix='tmp', directory=None, text=False):
     return fn
 
 
+@contextlib.contextmanager
```

Comment:
This is similar to `tempfile.NamedTemporaryFile`. Although that provides file-like objects